### ISSUE TITLE
Wrong Endpoint for Spot Get Active Order

### ISSUE THREAD

**Issue by creatzor** [bybit-exchange/pybit#15]
The endpoint for this function should be `/spot/v1/order` instead of `/spot/v1/history-orders`

https://github.com/bybit-exchange/pybit/blob/master/pybit/spot.py#L194

For the general HTTP class, it allows passing in an endpoint that could override this behavior. However, for the spot.HTTP class, there is no endpoint override.

**Comment by dextertd** [comment]
`get_active_order` in the other modules (`inverse_perpetual`, etc) is getting your non-conditional orders regardless of their `order_status` (whether they're open or closed). "Active" in the API document refers to a non-conditional order, not the state of the order (I know, it's confusing). So I think `/spot/v1/history-orders` is better for this method because at least it returns more than one order, whereas `/spot/v1/order` only returns a single order.

But I do think we could consider adding the `endpoint` arg to the `spot` module so that it matches the old compatibility version before the 2.0 restructuring.

I made a decision when [adding the spot endpoints to pybit](https://github.com/bybit-exchange/pybit/commit/a5ccc225b127452ae3f2f07d1d13fc487217a967) to keep the method names the same as the futures endpoints as much as possible, even though the functionality between them was not identical. This decision was made before the restructuring so I was forced to squeeze the new endpoints into the existing methods. Perhaps it could be changed in a future version but I don't think it's urgent. I think it's better to wait for future versions of the Bybit API.

**Comment by creatzor** [comment]
I see. In my case, I actually expected 1 order to be returned. I needed to get an order by id. Is there an alternative method to use for this?

Adding the endpoint optional arg would also make sense

**Comment by creatzor** [comment]
I ended up just manually sending my request for that endpoint.
Btw, is there any specific reason why USDC seems to be missing from this library? Just curious if it has to do with upcoming changes to the bybit api

**Comment by creatzor** [comment]
Using `/spot/v1/history-orders` trying to paginate and its weird. How can I get the history from start date to end date with a limit of 2?

I tried setting startTime to results[0].time+1 for the next fetch in the loop.. that didnt work.
I tried using orderId.. that didnt work..
I tried setting endTime to results[-1].time-1.. that didnt work.

I'm thoroughly confused as to how to paginate that. 

My endgoal with that is to simply fetch my filled trades from start time to end time. Any help or explanation of the parameters would be amazing
